searx/engines/pubmed.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+
+"""
+ PubMed (Scholar publications)
+
+ @website     https://www.ncbi.nlm.nih.gov/pubmed/
+ @provide-api yes (https://www.ncbi.nlm.nih.gov/home/develop/api/)
+
+ @using-api   yes
+ @results     XML
+ @stable      yes
+ @parse       url, title, publishedDate, content
+ More info on api: https://www.ncbi.nlm.nih.gov/books/NBK25501/
+"""
+
+from lxml import etree
+from datetime import datetime
+from searx.url_utils import urlencode
+try:
```

Comment:
Is it required beacuse of py2/py3 compatibility? If yes, please move it to `url_utils`, as it is where all proper URL functions are imported based on Python version.

Reply:
It is the case. It is quite usual to deal with urllib2 and urllib.request but as engines are not supposed to make requests by their own, it has not been used until now.